### DISCUSSION TITLE
JSON validation

### DISCUSSION THREAD

**Discussion by alstudi01** [Cardano-NFTs/policyIDs#5925] [Q&A]
I am trying to "**verify-json-validation**" ? 
I have to verify POLICY ID for NFT project on CNFT.io, I followed each and every step of README.md! every time JSON validation fails. I don't know what i am doing wrong. can anyone help me with this?

**Comment by MalSLim8** [comment]
I am new also, so still learning. When you run the json validator it should highlight in red the line that is incorrect. Change that line to match the example in readme.md

**Comment by specialnfts** [comment]
There is no red highlight on mine but still yellow dot appears.
